src/Core/src/AbstractApi.php

```diff
@@ -169,6 +182,11 @@ protected function getSignerFactories(): array
         ];
     }
 
+    protected function getAwsErrorFactory(): AwsErrorFactoryInterface
+    {
+        return new ChainAwsErrorFactory();
```

Comment:
The ChainAwsErrorFactory is only used as a fallback, correct?

Reply:
yes, this is for backward compatibility

Reply:
Cool cool 

Well done. Im really impressed by your work in your PRs

Reply:
thanks you :hugs: